applications/API/tests/test_endpoints.py

```
@@ -0,0 +1,287 @@
+            added_on=timezone.now(),
+        )
+        cls.i4 = Insult.objects.create(
+            content="Yo momma is so poor… Pending",
+            category=cls.cat_poor,
+            nsfw=True,
+            status=Insult.STATUS.PENDING,
+            added_by=cls.owner,
+            added_on=timezone.now(),
+        )
+
+        # Optional: an owner-only non-active insult to ensure it’s not leaked in public lists
```

Ruff (0.12.2)

Comment contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF003)